sixtyfps_compiler/builtins.60

```diff
@@ -187,6 +187,7 @@ export BoxShadow := _ {
 
 export struct Point := {
     //-name:sixtyfps::private_api::Point
+    //-rust_type_constructor:sixtyfps::re_exports::Point::new({})
```

Comment:
Do we actually still need that line since we handle this case in rust.rs ?

Reply:
I think you're right, we may not need this anymore.